Jenkinsfile

```diff
@@ -35,6 +35,7 @@ pipeline {
         GITHUB_USERNAME = 'codice'
         GITHUB_TOKEN = credentials('cxbot')
         GITHUB_REPONAME = 'alliance'
+        DOCKERHUB_CREDS = 'dockerhub-codicebot'
```

Comment:
Willing to give this a go. Wonder if we need another bot for Alliance specifically.